test/smoke/src/main.ts

```diff
@@ -279,7 +279,16 @@ after(async function () {
 	if (opts.log) {
 		const logsDir = path.join(userDataDir, 'logs');
 		const destLogsDir = path.join(path.dirname(opts.log), 'logs');
-		await new Promise((c, e) => ncp(logsDir, destLogsDir, err => err ? e(err) : c(undefined)));
+
+		// {{ SQL CARBON EDIT }}
+		/**
+		 * The logs directory is not present during the ADS web build, but is during the Darwin build.
+		 * In situations where the directory is missing and a copy attempt is made, bash exits with code 255 and raises an error
+		 * explaining that there's no such file or directory. This check prevents that error from occurring.
+		 */
+		if (fs.existsSync(logsDir)) {
```

Comment:
Let's just catch the error and log it instead - generally it's considered an anti-pattern to check for the existence of a file before attempting to create/delete it (because it's possible that it could be deleted/created inbetween the calls)